File: IKMS-server/src/main/java/pl/politechnika/ikms/rest/dto/register/PresenceDto.java
package pl.politechnika.ikms.rest.dto.register;

import lombok.Data;
import pl.politechnika.ikms.rest.dto.MinimalDto;

import java.time.LocalDateTime;

@Data
public class PresenceDto {

    private MinimalDto<Long, String> child;

    private boolean wasPresent;

    private MinimalDto<Long, String> checkingPerson;

    private LocalDateTime checkingTime;
}

File: IKMS-client/ngapp/src/app/employeeSiteModule/model/diary/presenceDto.ts
export class PresenceDto {
    child: any;
    wasPresent: boolean;
    checkingPerson: any;
    checkingTime: any;
    
    constructor() {
        this.child = null;
        this.wasPresent = false;
        this.checkingPerson = null;
        this.checkingTime = null;
    }
}